content/docs/1.6.0/high-availability/recover-volume-when-read-only.md

```diff
@@ -0,0 +1,13 @@
+---
+  title: Recover Volume When It Becomes Read Only
+  weight: 1
+---
+
+When an unexpected detachment happens, which can happen during a [Kubernetes upgrade](https://github.com/longhorn/longhorn/issues/703), a [Docker reboot](https://github.com/longhorn/longhorn/issues/686), or a network disconnection, the volume could become read only and leads to IO error.
```

Comment:
```suggestion
When an unexpected detachment happens
```

A volume is converted to read-only state after IO requests encountered IO errors rather than detachment.
IO errors can be caused by a network disconnection, high disk latency, abnormal node shutdown and so on.